.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - setup_remote_docker
       - run:
           name: Node 14 unit tests
-          command: cd nodejs && npm run ciTest
```

Comment:
TODO: need to update the coverage stages.

Also: shouldn't we run the tests on branches?

Reply:
Agreed that we should run tests on branches.